pkg/controller/apmserver/certificates/reconcile.go

```diff
@@ -20,12 +23,15 @@ import (
 )
 
 func Reconcile(
+	ctx context.Context,
 	driver driver.Interface,
 	apm *apmv1.ApmServer,
 	services []corev1.Service,
 	rotation certificates.RotationParams,
-) reconciler.Results {
-	results := reconciler.Results{}
+) *reconciler.Results {
+	span, _ := apmagent.StartSpan(ctx, "reconcile_certs", commonapm.SpanTypeApp)
+	defer span.End()
```

Comment:
As a general readability improvement, I think we should include a newline here by default.